modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb

```diff
@@ -63,27 +60,22 @@ def active
                        end
           if bgs_active.blank?
             message = "No Intent to file is on record for #{target_veteran_name} of type #{active_param}"
+            claims_v1_logging('itf_submit', message:)
```

Comment:
Should we add this logging line after line 52 as well, before raising the ResourceNotFound exception?